site_tests/tests/search-news-and-communications.spec.js

```diff
@@ -0,0 +1,54 @@
+const { test } = require('./fixtures');
+const { expect } = require('@playwright/test');
+
+test.describe('GOV.UK News and Communications Search Tests', () => {
+  test('search functionality - entering "example" shows expected results', async ({ page }) => {
+    await page.goto('/search/news-and-communications');
+
+    // Dismiss cookie banner if present
+    const rejectCookiesButton = page.getByRole('button', { name: 'Reject additional cookies' });
+    if (await rejectCookiesButton.isVisible()) {
+      await rejectCookiesButton.click();
+    }
+
+    // Enter 'example' in the main search input field (the finder search box)
+    const searchInput = page.locator('input[type="search"][name="keywords"]#finder-keyword-search');
+    await searchInput.fill('example');
+    await searchInput.press('Enter');
+
+    // Wait for the page to load the results
+    await page.waitForLoadState('networkidle');
+
+    // Verify the expected result text is visible
+    await expect(page.getByText('Illustrative example of a sample single line diagram')).toBeVisible();
+  });
+
+  test('filter functionality - selecting "Environment" taxon and "Updated (oldest)" shows expected results', async ({ page }) => {
+    await page.goto('/search/news-and-communications');
```

Comment:
This test timed out when running against prod:
```
    Error: page.waitForLoadState: Test timeout of 30000ms exceeded.

      34 |
      35 |     // Wait for page to fully load
    > 36 |     await page.waitForLoadState('networkidle')
```

Not ideal but we could triple the timeout: 
```suggestion
    test.slow();
    await page.goto('/search/news-and-communications');
```

Reply:
I tried it again a few times and it passes so let's leave it.